src/AcuityScheduling.js

```diff
@@ -2,60 +2,62 @@
  * AcuityScheduling Class
  */
 
-var request = require('request');
+var axios = require("axios");
 var pkg = require('../package');
 
 function AcuityScheduling (config) {
-
   config = config || {};
-
   this.base = config.base || AcuityScheduling.base;
   this.apiKey = config.apiKey;
   this.userId = config.userId;
-
   return this;
 }
 
 AcuityScheduling.base = 'https://acuityscheduling.com';
 AcuityScheduling.agent = 'AcuityScheduling-js/' + pkg.version;
 
 AcuityScheduling.prototype._request = function (path, options, cb) {
-
   options = options || {};
-  if (!cb) {
-    cb = (typeof options === 'function') ? options : function () {};
+  if (typeof options === 'function') {
+    cb = options;
   }
-
   path = typeof path === 'string' ? path : '';
+
   var config = {
     url: this.base + '/api/v1' + (path.charAt(0) === '/' ? '' : '/') + path,
-    json: true
   };
 
   // Set configurable options:
   if (options.auth)     config.auth     = options.auth;
-  if (options.body)     config.body     = options.body;
+  if (options.data)     config.data     = options.data;
   if (options.method)   config.method   = options.method;
-  if (options.qs)       config.qs       = options.qs;
```

Comment:
We should keep this for backwards compatibility, too.  Looks fine to map it to the `params` option for axios?